tests/unit/test_charm.py

```diff
@@ -168,9 +175,10 @@ def test_update_status(harness, lk_manifests_client, lk_charm_client):
     harness.begin()
 
     # With nothing mocked, all resources will appear as missing
+    harness.charm._stored.configured = True
+    harness.charm.model.unit.status = expected = BlockedStatus("unchanged on missing")
```

Comment:
Unchanged on missing? Is that the best wording?

Reply:
i just tried to make this more clear.  I just wanted some text that clearly indicates the charm code isn't changing the blocked message